src/hooks/useFindResourceOwner/index.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import { useState, useEffect } from "react";
+import { useSession } from "@inrupt/solid-ui-react";
+import { getStringNoLocale } from "@inrupt/solid-client";
+import { foaf, vcard } from "rdf-namespaces";
+import { joinPath } from "../../stringHelpers";
+import { getResource } from "../../solidClientHelpers/resource";
+
+export default function useFindResourceOwner(iri) {
+  const session = useSession();
+  const { fetch } = session;
+  const [ownerName, setOwnerName] = useState();
+  const [error, setError] = useState();
+  const resourceOrigin = new URL(iri).origin;
+  const profileIri = joinPath(resourceOrigin, "profile/card#me");
```

Comment:
this is the bit that's still TBC, right? This is fine for the web IDs we're likely to use for the time being, but others won't necessarily match this

Reply:
Yes, this and we might have the WebID stored in the bookmarks so we won't have to do all of this concatenating to get it from the resource.

Reply:
This will also break on ESS, which uses subdirectories for pod origins (although it'll be fixed after GA, and we'll have the ownership API implemented before that.)